projects/Mallard/src/components/front/index.tsx

```diff
@@ -165,11 +165,13 @@ const FrontWithResponse = React.memo(
                     // These three props are responsible for the majority of
                     // performance improvements
                     initialNumToRender={2}
-                    windowSize={5}
+                    windowSize={3}
```

Comment:
Did you mean to change this @walaura?

Reply:
If we _do_ turn off renderClippedSubviews, then this shouldn't be needed right? I've found that a window size of 3 leaves a few empty holes on slower Androids but not necessarily the end of the world.

Reply:
at this stage I wanna go as low as possible because I'm terrified of crashes tbh